resource-agents.spec.in

```diff
@@ -101,7 +101,9 @@ Requires: /usr/sbin/mount.cifs
 %else
 Requires: /sbin/fsck
 Requires: /sbin/fsck.ext2 /sbin/fsck.ext3 /sbin/fsck.ext4
+%if 0%{?rhel} == 0
```

Comment:
Since if condition is nested, I think the message is roughly correct. excerpts from a https://github.com/ClusterLabs/resource-agents/commit/32ddaf22548d3021773127d4a8abb1df77179450
``` rb
 96    %if 0%{?centos_ver} > 6 || 0%{?rhel} > 6
 :         # run on RHEL/CentOS 7 or later
101    %else
 :         # run on RHEL/CentOS 6 or earlier and other distributions
104    %if 0%{?rhel} == 0
 :         # run on a distribution other than RHEL/CentOS
105    Requires: /sbin/fsck.xfs
106    %endif
 :
108    %endif
```